session/session.go

```diff
@@ -1436,9 +1440,99 @@ func (s *session) ExecuteStmt(ctx context.Context, stmtNode ast.StmtNode) (sqlex
 		}
 		return nil, err
 	}
+	if !s.isInternal() && config.GetGlobalConfig().EnableTelemetry {
+		CountTelemetry(s)
+	}
 	return recordSet, nil
 }
 
+// CountTelemetry records the telemetry.
+func CountTelemetry(s *session) {
+	telemetry.CoprocessorCacheTelemetry.Lock.Lock()
+	length := len(telemetry.CoprocessorCacheTelemetry.MinuteWindow)
+	if length == 0 || time.Since(*telemetry.CoprocessorCacheTelemetry.MinuteWindow[length-1].BeginAt) >= time.Minute {
+		var i int
+		for i = 0; i < length && time.Since(*telemetry.CoprocessorCacheTelemetry.MinuteWindow[i].BeginAt) >= 6*time.Hour; i++ {
```

Comment:
May use some meaning for definition to instead of 6, such as  TELEMETRY_UPDATE_INTERVAL ?